services/time/tests/test_exonum_time.rs

```diff
@@ -24,19 +25,35 @@ fn assert_storage_times_eq<T: AsRef<Snapshot>>(
 ) {
     let schema = TimeSchema::new(snapshot);
 
-    assert_eq!(schema.time().get().map(|time| time), expected_current_time);
+    assert_eq!(schema.time().get(), expected_current_time);
 
     let validators_times = schema.validators_times();
     for (i, validator) in validators.iter().enumerate() {
         let public_key = &validator.public_keys().service_key;
 
         assert_eq!(
-            validators_times.get(public_key).map(|time| time),
+            validators_times.get(public_key),
             expected_validators_times[i]
         );
     }
 }
 
+fn assert_transaction_result(
+    result: Option<TransactionResult>,
+    code: u8,
+    description: Option<&str>,
```

Comment:
As far as I see, this is never used in tests. Imo, it could make more sense to *return* `Option<&str>` on success in order to be able to check it with various `str` methods.